realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -111,6 +111,9 @@ BaseRealSenseNode::BaseRealSenseNode(rclcpp::Node& node,
     _is_initialized_time_base(false),
     _camera_time_base(0),
     _sync_frames(SYNC_FRAMES),
+    _enable_rgbd(ENABLE_RGBD),
```

Comment:
I didn't see where you use the boolean value given by the user and replaces this constant, but I assume it exists somewhere. please show me.

Reply:
https://github.com/IntelRealSense/realsense-ros/blob/0f2aa7c1b68bba626d9e1fcf8854238f797c5047/realsense2_camera/src/parameters.cpp#L49